specs/jsonschema-core.md

```diff
@@ -415,6 +415,16 @@ keywords MUST NOT begin with this prefix.
 Implementations MUST refuse to evaluate schemas which contain keywords which
 they do not know how to process or explicitly choose not to process.
 
+## Specification Versioning and Compatibility
+
+This specification is identified collectively by two values: version and release
+year.
+
+A schema written to conform with the requirements of a given version is
```

Comment:
```suggestion
A schema written to conform with the requirements of a given release is
```

Reply:
I think this could go either way.  Is there a compelling reason you think it should be "release"?

Reply:
"Release" makes perfect sense to me, but I'm having trouble convincing myself one way or another that "version" makes sense.

I don't think it makes sense to write a schema to conform with the requirements of a given version. If there are five releases for that version, which one are you writing your schema for? Saying I'm writing a v1 schema doesn't tell me if I can use a keyword introduced in 2026 (assuming there's also a 2025). The schema author is always has to be writing toward a release.

From the perspective of a validator, things are a little different. The validator would validate against a version and the release doesn't matter so much. But, this is a statement about the schema, not the validator.